build.gradle

```diff
@@ -0,0 +1,274 @@
+plugins {
+    id 'java'
+    id 'org.springframework.boot' version '2.7.12'
+    id 'io.spring.dependency-management' version '1.0.15.RELEASE'
+    id 'eclipse'
+    id 'checkstyle'
+    id 'com.diffplug.spotless' version '6.19.0'
+    id 'net.ltgt.errorprone' version '3.1.0'
+}
+
+group = 'org.mifos.connector'
+version = '0.0.1-SNAPSHOT'
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(17)
+    }
+}
+
+configurations {
+    compileOnly {
+        extendsFrom annotationProcessor
+    }
+}
+ext {
+    zeebClientVersion = '8.1.23'
+    springBootVersion = '2.7.12'
+}
+
+repositories {
+    mavenCentral()
+    maven {
+        url = uri('https://jfrog.sandbox.fynarfin.io/artifactory/fyn-libs-snapshot')
+    }
+}
+
+dependencies {
+    implementation 'org.springframework.boot:spring-boot-starter-actuator'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    compileOnly 'org.projectlombok:lombok'
+    annotationProcessor 'org.projectlombok:lombok'
+    testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
+    implementation 'org.mifos:ph-ee-connector-common:1.9.1-SNAPSHOT'
+    implementation 'org.json:json:20210307'
+    checkstyle 'com.puppycrawl.tools:checkstyle:10.9.3'
+    checkstyle 'com.github.sevntu-checkstyle:sevntu-checks:1.44.1'
```

Comment:
Add openapi dependency if in case you add apis

Reply:
Done.

Reply:
resolve this conversation